habitat-baselines/habitat_baselines/config/habitat_baselines/rl/policy/hab3_hl_neural_ma.yaml

```diff
@@ -0,0 +1,31 @@
+name: "HierarchicalPolicy"
+obs_transforms:
+  add_virtual_keys:
+    virtual_keys:
+      "goal_to_agent_gps_compass": 2
+hierarchical_policy:
+  high_level_policy:
+    name: "NeuralHighLevelPolicy"
+    allowed_actions:
+      - nav
+      - pick
+      - place
+      - nav_to_receptacle_by_name
+    allow_other_place: False
+    hidden_dim: 512
+    use_rnn: True
+    rnn_type: 'LSTM'
+    backbone: resnet18
+    normalize_visual_inputs: False
+    num_rnn_layers: 2
+    replan_dist: -1.0
+    policy_input_keys:
+      - "head_depth"
```

Comment:
Could we put this somewhere else? Maybe create a config wihtout the policy_input_keys and one that extends it with the input_keys. Otherwise I need to separate a fully new config for the humanoid. 
Alternatively, we could copy these into the hab3_experiment configs, since we are putting `gym.obs_keys` there.

Reply:
Agreed. Done.